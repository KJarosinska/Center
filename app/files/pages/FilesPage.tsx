import * as React from 'react';
import { RouteComponentProps } from 'react-router';
import styled from "styled-components";
import Files from "../components/Files";

const FilePageWrapper = styled.div`

`;

export class FilesPage extends React.Component<RouteComponentProps<{}>, {}> {
  render() {
    return (
      <FilePageWrapper>
        <Files/>
      </FilePageWrapper>
    );
  }
}

export default FilesPage;
