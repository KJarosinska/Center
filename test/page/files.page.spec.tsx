import { shallow, ShallowWrapper } from 'enzyme';
import 'jest-styled-components'
import * as React from "react";
import FilesPage from '../../app/files/pages/FilesPage';

let wrapper: ShallowWrapper;
const FilesPageAny = FilesPage as any;

describe('files page', () => {

  beforeEach(() => {
    wrapper = shallow(<FilesPageAny/>);
  });

  it('should match snapshot', () => {
    expect(wrapper.dive()).toMatchSnapshot();
  });

});