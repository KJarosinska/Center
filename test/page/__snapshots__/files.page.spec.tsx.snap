// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`files page should match snapshot 1`] = `
<div
  className=""
>
  <Connect(Files) />
</div>
`;
